tutorial_sarscov2.md

```diff
@@ -0,0 +1,231 @@
+---
+layout: default
+permalink: /tutorial/sars-cov2/
+---
+
+This tutorial shows you the basic of how to interact with V-pipe.
+
+For the purose of this Tutorial, we will work with the `sars-cov2` branch which is [adapted for the SARS-CoV-2 virus]({{ "/sars-cov-2/" | relative_url }}).
+
+> **Organisation of Data** :
+>
+> V-pipe expects its samples data organized in a two-level hierarchy:
```

Comment:
maybe better: V-pipe expects the input samples to be organized in a two-level hierarchy

Reply:
And I dont think the link is overly helpful here, maybe remove it or name the full directory structure?

Reply:
Didn't got this last comment? What edits are you suggesting?

My idea is to point to the page of the wiki manual where Susia has gone to great lenght to document everything.

Reply:
Susia means dirty in spanish :laughing: